ecf/hurricane/plots/jevs_hurricane_regional_tropcyc_plots.ecf

```diff
@@ -43,14 +43,15 @@ export STEP=plots
 export envir=dev
 export cyc=00
 export job=jevs_hurricane_regional_tropcyc_plots_${cyc}
+export PDY=20221231 #Set PDY to override setpdy.sh called in the j-jobs
 
 #Define the directory for TC-stats file 
-export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/1.0/hurricane_regional/tropcyc/stats
+export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/v1.0/hurricane_regional/tropcyc/stats
```

Comment:
Let's utilize the version settings in `run.ver`. Please use ${evs_ver} in place of "v1.0" in COMINstats definition.